src/components/RadioConditionalInput.tsx

```diff
@@ -360,14 +366,21 @@ const renderRadioButtonSet = (radio: RadioButton, dates?: ProductDateInformation
         </label>
     );
 
+    const radioButtonHint: ReactElement = (
+        <span className="govuk-hint govuk-radios__hint" id={radio.radioButtonHint?.id}>
+            {radio.radioButtonHint?.content}
+        </span>
+    );
+
     if (isRadioWithConditionalInputs(radio)) {
-        return renderConditionalRadioButton(radio, radioButtonLabel, dates);
+        return renderConditionalRadioButton(radio, radioButtonLabel, radioButtonHint, dates);
     }
 
     return (
         <div key={radio.id} className="govuk-radios__item">
             <input className="govuk-radios__input" id={radio.id} name={radio.name} type="radio" value={radio.value} />
             {radioButtonLabel}
+            {radioButtonHint}
```

Comment:
Do we want this `radioButtonHint` to only be rendered conditionally? If a fieldset doesn't have a radioButtonHint attribute (e.g. with defineTimeRestrictions and definePassengerType), I'm not sure we want an empty span being rendered?